### ISSUE TITLE
Include examples in entroq package documentation

### ISSUE THREAD

**Issue by shiblon** [shiblon/entroq#14]
Opening up the entroq package documentation doesn't tell you how to use it at all. That needs to change. It's a nontrivial thing to use.

**Comment by lwileczek-dmc** [comment]
Potentially OBE?

**Comment by shiblon** [comment]
No, I think it still needs some significant help, honestly. :-P
